tests/plugins/shift_enter_paste_into_list/extrapastefilter.js

```diff
@@ -0,0 +1,41 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: toolbar,clipboard,pastetext,shift_enter_paste_into_list */
+/* bender-include: ../clipboard/_helpers/pasting.js */
+/* global assertPasteEvent */
+
+( function() {
+	'use strict';
+	bender.editor = {
+		config: {
+			enterMode: CKEDITOR.ENTER_P,
+			shiftEnterMode: CKEDITOR.ENTER_BR
+		}
+	};
+
+	var tests = {};
+
+	tests.setUp = function() {
+		// be inside a list item
+		bender.tools.setHtmlWithSelection( this.editor, '<ol><li>^</li></ol>' );
+	};
+
+	tests[ 'test_single_p_get_removed_in_list_item' ] = function() {
+		var editor = this.editor,
+			pasteContent = '<p>asdf</p>',
+			expectedContent = 'asdf';
+
+		assertPasteEvent( editor, { dataValue: pasteContent },
```

Comment:
this is an assertion helper used in some of the existing clipboard tests, basically checking only the pasted content for changes.